src/NativeScript/ObjC/IsObjcObject.cpp

```diff
@@ -214,90 +36,13 @@ bool IsObjcObject(const void* inPtr) {
         return false;
     }
 
-#if !defined SKIP_OBJECTIVE_C_CHECKS || !SKIP_OBJECTIVE_C_CHECKS
-    //
-    // Check for tagged pointers
-    //
-    if (IsObjcTaggedPointer(inPtr, NULL)) {
-        return true;
-    }
-#endif
-
-    //
-    // Check if the pointer is aligned
-    //
-    if (((uintptr_t)inPtr % sizeof(uintptr_t)) != 0) {
```

Comment:
The check could also be kept as it doesn't use any forbidden APIs and can serve at least as an optimization.

Reply:
This was also giving me trouble with the above case with `127`/`NSNumber`